gulpfile.js

```diff
@@ -156,6 +159,7 @@ createTask('e2e', e2e);
 createTask('firebase', firebase);
 createTask('get-zindex', getZindex);
 createTask('integration', integration);
+createTask('init-bento-component', initBentoComponent);
```

Comment:
Usability nit: The gulp task that creates boilerplate for a new extension is called `make-extension`. Maybe call this task `make-bento-component` for uniformity? (I have no strong preference here.)

Reply:
Ahhh I had looked for `init` or `create`, totally missed the `extension-generator` folder. I'll rename it to `make-bento-component` unless someone in bento objects. Thanks!

Reply:
I know the template files are rather different, but conceptually I'd categorize both the existing task and this new task as both "making extensions." Is it possible at all to give the existing `make-extension` a `--bento` flag instead of creating `make-bento-extension`? Might make it more findable. But I don't feel strongly about this.

Reply:
Definitely. In the feature request issue I had imagined `gulp init-component --bento` but didn't find the original task. I'll rework this into that task; holding off on that until the minor tweaks/fixes are in place to minimize churn on the files here

Reply:
Done! `gulp make-extension --bento --name=foo-bar`